src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -86,6 +86,7 @@
         "symfony/mime": "<4.4",
         "symfony/property-info": "<4.4",
         "symfony/property-access": "<5.3",
+        "symfony/runtime": "<5.4.45|>=6.0,<6.4.13|>=7.0,7.1.6",
```

Comment:
ensures that we symfony/runtime is installed, it's a version that always registers ErrorHandler